ui/packages/statement/src/components/StatementDetail.tsx

```diff
@@ -116,7 +116,7 @@ export default function StatementDetail({
             <div style={{ marginTop: 6 }}>
               <h3>{t('statement.plan.plans')}</h3>
               <div className={styles.table_wrapper}>
-                {detail.plans!.map((plan) => (
+                {detail.plans!.map(plan => (
```

Comment:
You should keep the parentheses else you can't pass the CI. we use the latest prettier 2.x version, it will add the parentheses for the arrow function one parameter situation.